docs/README.md

```diff
@@ -1,4 +1,4 @@
 README
 ======
 
-This file is a placeholder. This directory will contain public documentation.
+This file is a placeholder.  This directory will contain public documentation.
```

Comment:
I thought we preferred one space after periods now? We may need to publish a documentation style guide... :)